README.md

```diff
@@ -2243,6 +2244,53 @@ department_with_peter = departments.values(is_equal_to("people[any].name", "Pete
 
 `HazelcastJsonValue` is a lightweight wrapper around your JSON strings. It is used merely as a way to indicate that the contained string should be treated as a valid JSON value. Hazelcast does not check the validity of JSON strings put into to the maps. Putting an invalid JSON string into a map is permissible. However, in that case whether such an entry is going to be returned or not from a query is not defined.
 
+#### 7.7.1.5. Filtering with Paging Predicates
+
+The Python client provides paging for defined predicates. With its `PagingPredicate` class, you can get a list of keys, values, or entries page by page by filtering them with predicates and giving the size of the pages. Also, you can sort the entries by specifying comparators.
+
+In the example code below:
+
+The `greater_equal_predicate` gets values from the "students" map. This predicate has a filter to retrieve the objects with an "age" greater than or equal to 18.
+
+Then a `PagingPredicate` is constructed in which the page size is 5, so that there are five objects in each page. The first time the values are called creates the first page.
+
+It gets subsequent pages with the `next_page()` method of `PagingPredicate` and querying the map again with the updated `PagingPredicate`.
+
+```python
+from hazelcast.serialization.predicate import is_greater_than_or_equal_to, PagingPredicate
+from hazelcast import HazelcastClient
+
+student_map = HazelcastClient().get_map('students').blocking()
+greater_equal_predicate = is_greater_than_or_equal_to('age', 18)
+paging_predicate = PagingPredicate(greater_equal_predicate, 5)
```

Comment:
instead of directly instantiating  `PagingPredicate` could you use the `paging_predicate` method to make it inline with other usages in the documentation?